client/Elements/Components/Topics/TopicsList.patternfly.tsx

```diff
@@ -120,9 +121,8 @@ export const TopicsList: React.FunctionComponent<ITopicList> = ({
 
   return (
     <>
-      <Title headingLevel='h2' size='lg'>
-        Topics
-      </Title>
+     <Page>
+       <PageSection>
```

Comment:
Since we can use either Page or PageSection at top level, we should remove this fragment too.
Do we really need a `Page` or we can go ahead with just `PageSection`.

Reply:
The view matches the design better with ```PageSection``` wrapped inside a ```Page```

Reply:
Once we have #58 merged, we will have a `Page` at top level already, then we should remove `Page` from here.